### PR TITLE
[IOTDB-2430] Add example of revoke Privilege

### PR DESCRIPTION
**Pull request by tsunghanjacktsai**
## Description

Issue: [[IOTDB-2430] Add example of revoke Privilege](https://issues.apache.org/jira/browse/IOTDB-2430?jql=project%20%3D%20IOTDB%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20in%20(easy-fix%2C%20easy-fixed%2C%20EasyFix%2C%20easyfix%2C%20easytest%2C%20easytask%2C%20easytext)%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC)

I was actually not sure if I should add single or double quote string literals for the privilege names (e.g. revoke user sgcc_write_user privileges 'INSERT_TIMESERIES' on root.sgcc). 

I did see the string literals for INSERT_TIMESERIES in the documents of the previous versions, but I didn't see the string literals in the latest version, which I think is weird. So I just left it the same as the current version.

<hr>

This PR has:
- [x] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [x] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

**Comment by SteveYurongSu** [comment]
Hi @jack870131, nice to see you back!

**Comment by jack870131** [comment]
> Hi @jack870131, nice to see you back!

Hey, sorry for going without contributions for an year. Nice to see the community again!

File: docs/zh/UserGuide/Administration-Management/Administration.md [hunk 1]
```diff
@@ -113,6 +113,27 @@ IoTDB> INSERT INTO root.ln.wf01.wt01(timestamp, status) values(1509465600000, tr
 Msg: The statement is executed successfully.
 ```
 
+### 撤销用户权限
+
+授予用戶權限後，我們可以使用 `REVOKE USER <userName> PRIVILEGES <privileges> ON <nodeName>` 來撤銷已授予的用戶權限。 例如：
```
Comment (HTHou) [inline docs/zh/UserGuide/Administration-Management/Administration.md:118]: Should we use the simplified Chinese here to keep the consistency. :)

Comment (tsunghanjacktsai) [reply docs/zh/UserGuide/Administration-Management/Administration.md:118]: Oh, sorry for this. I'll correct it.

**Review by SteveYurongSu — APPROVED** [review]
Thanks for your contribution!!!

**Review by SteveYurongSu — APPROVED** [review]
Thanks for your contribution!!!

**Pull request merged**